script/Arcueid/Day 6/06_00_ARC06_3_2.txt

```diff
@@ -378,8 +378,8 @@ I reply to the man clad head to toe in bandages.
 [sha:f332fce4ddc3cbfbdb6f369bf930c60d2c6b3d89]{
 -- Page 302, Offset 14654.
 -- ……しかし、そうだな。過去から戻ってきたとしても、見違えるような現実など、そうはないという事か」
-Still, true enough...
- I suppose that even if you were to dredge such things up from your past, you wouldn't mistake them for reality."
+Still, I suppose so.
+　Even if you've come back from the past, there isn't much of reality you'd consider out of place."
```

Comment:
I talked to a friend who has played the game and we talked about this line a bit.
It's still really cryptic, but this is probably closer to what is meant? Don't wanna mess this up, but I'm not sure if it's possible to understand until Red Garden.